pkg/util/cert.go

```diff
@@ -72,6 +72,43 @@ func GenCA(common string, svcName []string, duration time.Duration) (*rsa.Privat
 	return key, ca, err
 }
 
+func LoadRootCA(dir, certFilename, keyFilename string) error {
+	key, err := ioutil.ReadFile(filepath.Join(dir, keyFilename))
+	if err != nil {
+		return errors.Wrap(err, "error reading CA key")
+	}
+
+	block, _ := pem.Decode(key)
+	if block == nil {
+		return errors.New("failed to parse PEM block containing the CA key")
+	}
+
+	rootKey, err = x509.ParsePKCS1PrivateKey(block.Bytes)
+	if err != nil {
+		return errors.Wrap(err, "parsing CA key from PEM")
+	}
+
+	certPath := filepath.Join(dir, certFilename)
+	cert, err := ioutil.ReadFile(certPath)
+	if err != nil {
+		return errors.Wrap(err, "reading CA certificate")
+	}
+
+	block, _ = pem.Decode(cert)
+	rootCA, err = x509.ParseCertificate(block.Bytes)
```

Comment:
util.cert has a function called PemToCertificate that might be good to use.

Reply:
makes sense

Reply:
Done.